test/webapp-strategy-test.js

```diff
@@ -538,41 +540,41 @@ describe("/lib/strategies/webapp-strategy", function () {
 				done();
 			};
 			webAppStrategy.success = function (user) {
-				
+
 				assert.equal(options.successRedirect, "redirectUri");
 				assert.isObject(req.session[WebAppStrategy.AUTH_CONTEXT]);
-				
+
 				assert.isString(req.session[WebAppStrategy.AUTH_CONTEXT].accessToken);
 				assert.equal(req.session[WebAppStrategy.AUTH_CONTEXT].accessToken, "access_token_mock");
 				assert.isObject(req.session[WebAppStrategy.AUTH_CONTEXT].accessTokenPayload);
 				assert.equal(req.session[WebAppStrategy.AUTH_CONTEXT].accessTokenPayload.scope, "appid_default");
-				
+
 				assert.isString(req.session[WebAppStrategy.AUTH_CONTEXT].identityToken);
 				assert.equal(req.session[WebAppStrategy.AUTH_CONTEXT].identityToken, "id_token_mock");
 				assert.isObject(req.session[WebAppStrategy.AUTH_CONTEXT].identityTokenPayload);
 				assert.equal(req.session[WebAppStrategy.AUTH_CONTEXT].identityTokenPayload.scope, "appid_default");
-				
+
 				assert.isObject(user);
 				assert.equal(user.scope, "appid_default");
-				
+
 				done();
 			};
-			
+
 			var req = {
 				session: {},
 				query: {
 					code: "WORKING_CODE",
 					state: ""
 				}
 			};
-			
+
 			var options = {
 				successRedirect: "redirectUri"
 			};
 			req.session[WebAppStrategy.STATE_PARAMETER] = { anonymousLogin : false , state : "123456789" };
 			webAppStrategy.authenticate(req, options);
 		});
-		
+
 		it("Should handle callback if request contains grant code. Success with original URL", function (done) {
```

Comment:
Add a test in here for webappstrategy should fail if the verify callback is called with an error